src/modules/files-storage/api/controller/api-test/wopi.api.spec.ts

```diff
@@ -668,4 +672,232 @@ describe('Wopi Controller (API)', () => {
 			});
 		});
 	});
+
+	describe('putFile', () => {
+		describe('when file is uploaded successfully', () => {
+			const setup = async () => {
+				const fileRecord = fileRecordEntityFactory.buildWithId();
+				const accessToken = accessTokenResponseTestFactory().build().token;
+				const query = wopiAccessTokenParamsTestFactory().withAccessToken(accessToken).build();
+				const wopiPayload = wopiPayloadTestFactory().withFileRecordId(fileRecord.id).withCanWrite(true).build();
+				const accessTokenPayloadResponse = accessTokenPayloadResponseTestFactory().withPayload(wopiPayload).build();
+
+				authorizationClientAdapter.resolveToken.mockResolvedValueOnce(accessTokenPayloadResponse);
+
+				jest.spyOn(FileType, 'fileTypeStream').mockImplementation((readable) => Promise.resolve(readable));
+
+				await em.persistAndFlush(fileRecord);
+
+				fileStorageConfig.FEATURE_COLUMN_BOARD_COLLABORA_ENABLED = true;
+
+				return { fileRecord, query };
+			};
+
+			it('should return 200 and updated file record', async () => {
+				const { fileRecord, query } = await setup();
+
+				const response = await testApiClient
+					.post(`/files/${fileRecord.id}/contents`)
+					.query(query)
+					.attach('file', Buffer.from('abcd'), 'test.txt');
+
+				const updatedFileRecord = await em.findOne(FileRecordEntity, fileRecord.id);
+
+				expect(response.status).toBe(200);
+				expect(response.body.LastModifiedTime).toBe(updatedFileRecord?.contentLastModifiedAt?.toISOString());
+			});
+		});
+
+		describe('when WOPI feature is disabled', () => {
+			const setup = () => {
+				const fileRecord = fileRecordEntityFactory.buildWithId();
+				const query = wopiAccessTokenParamsTestFactory().build();
```

Comment:
Ich würde hier eine vollständiges setup wie bei "when file is uploaded successfully" erwarten.
Was wäre den wenn wir das setup für beide gleich machen und das feature flag true/false in das it ziehen. 

Nicht ganz Regelkonform, aber ein sehr viel belastbarer Test für feature is disabled.